torchvision/transforms/v2/functional/_utils.py

```diff
@@ -47,6 +47,11 @@ def _name_to_dispatcher(name):
 
 
 def register_kernel(dispatcher, datapoint_cls):
+    """Register a kernel for a dispatcher and a (custom) datapoint type.
```

Comment:
We should at least mention that this is meant as a decorator or provide a minimal code snippet. I know this is explained in the tutorial, but I would prefer to have something here as well.

Reply:
I had that originally, then I realized this was **exactly** the same code as the one in the gallery example. I don't mind adding it back if you think it's worth it, but it will be redundant.

And since those docstrings examples are never executed (although technically we could have that), we wouldn't be able to catch it if there's ever an issue in that code

Reply:
Maybe just

```suggestion
    """Decorate a kernel to register it for a dispatcher and a (custom) datapoint type.
```

?